content/rule_book/quiz_events.md

```diff
@@ -63,7 +63,7 @@ Limited to five verses before or after the verse. On all question types, a quizz
 
 The quizzer's response is correct when:
 
-- It fulfills all requirements specific to the Question Type.
+- It fulfills all requirements specific to the question type.
```

Comment:
Question Type is a defined term, so will be denoted somehow, with Italics @gryphonshafer ?

Reply:
I was working backwards, so I forgot that is was defined. I would leave it capital as it has become a proper noun of sorts

Reply:
yeah I don't care what we do, but it should be uniform throughout. I was just making a stab by making it caps.

Reply:
I've changed it back to Question Type

Reply:
Don't worry about exact term syntax right now. I'll cover that in my first OCD pass of phase 2.